collect_app/src/main/res/values/strings.xml

```diff
@@ -482,4 +482,6 @@
 <string name="exit">Exit</string>
 <string name="instance_scan_completed">Finished scanning.</string>
 <string name="instance_scan_count">%1$d forms added.</string>
+<string name="instance_sync">Sync form instance</string>
```

Comment:
I try not to use instance because it's not used anywhere else, really. I think maybe `Import saved forms as...`. Then the summary would be `Imports saved forms from instances folder`